### DISCUSSION TITLE
Dock turn off in lawnchair 13 +

### DISCUSSION THREAD

**Discussion by TheBlunius** [LawnchairLauncher/lawnchair#3860] [Q&A] [answered]
Will there be an option to remove/disable dock in lawnchair 13?
+is there avaible lawnchair 13 in dev version to install?

**Comment by MrSluffy** [comment]
It will be implemented soon

**Comment by validcube** [comment, marked as answer]
Implemented in Lawnchair 14 — afb5e371461e02d1acd995f9c4bb7a2a31632130
